### PR TITLE
298407 query api

### PR DESCRIPTION
**Pull request by jbarnsley10**

### CODE REVIEW

File: test/integration/narrow/routes/events.test.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    const options = {
+      method: 'GET',
+      url: '/events'
+    }
+
+    const response = await server.inject(options)
+
+    expect(response.statusCode).toBe(400)
+  })
+
```
Comment (whitewaterdesign) [inline test/integration/narrow/routes/events.test.js:71]: Would it be possible to have a test for multiple pks' as well? ``/events?pks=ED1,ED2``

File: README.md [hunk 2]
```diff
@@ -125,6 +125,27 @@ The script accepts the following arguments:
 - `--watch/-w` - run tests with file watching to support Test Driven Development scenarios (TDD)
 - `--debug/-d` - run tests in debug mode. Same as watch mode but will wait for a debugger to be attached before running tests.
 
+
+process.env.AZURE_STORAGE_ACCOUNT_NAME = 'devstoreaccount1'
```
[the remark this answers is not in the document]
Comment (jbarnsley10) [reply README.md:129]: Yes, this is the default name when using Azurite in your local dev. Even the secret key is the same for everyone so even not a problem to save secret key (although SonarCloud may flag as it being a secret)

File: test/integration/narrow/routes/events.test.js (hunk 1 above)
Comment (jbarnsley10) [reply test/integration/narrow/routes/events.test.js:71]: Yes, I forgot that. Good spot, I'll add one

**Review by whitewaterdesign — APPROVED** [review]
Good stuff

**Pull request merged**